Sources/Quick/Filter.swift

```diff
@@ -22,9 +22,17 @@ final public class Filter: NSObject {
     }
 
     /**
-        Example and example groups with [Pending: true] are excluded from test runs.
+        Example and example groups with [Excluded: true] are excluded from test runs.
         Use this to temporarily suspend examples that you know do not pass yet.
     */
+    public class var excluded: String {
+        return "excluded"
+    }
+
+    /**
+     Example and example groups with [Pending: true] are pending from test runs.
+     Use this to temporarily suspend examples that you know do not pass yet.
+     */
```

Comment:
nit-pick: the indentation for the documentation here doesn't match the indentation in the rest of this repository. It should be formatted like this:

```swift
/**
    Example and example groups with [Pending: true] are pending from test runs.
    Use this to temporarily suspend examples that you know do not pass yet.
*/
```